api/src/controllers/listing.controller.ts

```diff
@@ -76,15 +76,16 @@ export class ListingController {
     private readonly listingCsvExportService: ListingCsvExporterService,
   ) {}
 
-  @Get()
+  @Post('list')
```

Comment:
todo(non-blocking): Since this is such an important endpoint I think we should keep it in sync with core as much as possible. We don't need to do it as part of this fix, but ideally we would also make this change in core if this is the pattern we want to go with

Reply:
Will make a ticket!

Reply:
https://github.com/bloom-housing/bloom/issues/4515